src/app.rs

```diff
@@ -3763,13 +3767,22 @@ pub fn remove_tag(task: &mut Task, tag: &str) {
 }
 
 #[cfg(test)]
+// Disabled, as "'" should be a String for more readable shlex shell escaping.
+#[allow(clippy::single_char_pattern)]
 mod tests {
-  use std::{ffi::OsStr, fmt::Write, fs::File, io, path::Path};
+  use std::{ffi::OsStr, fmt::Write, fs::File, io, path::{Path, PathBuf}};
 
   use ratatui::{backend::TestBackend, buffer::Buffer};
 
   use super::*;
```

Comment:
The second, potentially more controversial one(as I don't know if there was specific reasoning for compile time embedding).

Mostly a minor improvement, intended so that the different TASKDATA can be specified for the tests without a recompile.
It has the additional free benefit that it avoids 4 compiler warnings if the developers chosen IDE needs additional work to start rust-analzyer in the direnv context(e.g. VSCode requires the direnv extension. It is not required for the inbuilt terminal, so VSCode seems to do some shell magic under the hood).